cfgov/cfgov/settings/__init__.py

```diff
@@ -0,0 +1 @@
+from .local import *
```

Comment:
hmm but wouldn't this force it to use local settings every-time? where would be the flip for base/prod settings?